packages/frontity-org-theme/src/components/lazy-video.tsx

```diff
@@ -0,0 +1,22 @@
+import useInView from "@frontity/hooks/use-in-view";
```

Comment:
I would disable the rule `jsx-a11y/media-has-caption` for this file, using the following line at the top:
```ts
/* eslint-disable jsx-a11y/media-has-caption */
```

Reply:
Done, thanks David :slightly_smiling_face: